src/integers.jl

```diff
@@ -0,0 +1,50 @@
+# Int32, Int64 constructors handled by Cxx side
+Polymake.pm_Integer(int::Int128) = Polymake.pm_Integer(big(int))
+Polymake.pm_Integer(int::BigInt) = Polymake.cast_bigint_to_pm_integer!(copy(int))
+Polymake.pm_Integer(int::Integer) = Polymake.pm_Integer(convert(BigInt, int))
+
+"""
+    bigint!(int::Polymake.pm_IntegerAllocated)
+
+Cast a `Polymake.pm_IntegerAllocated` to a `BigInt`. This *does not* create a copy.
+"""
+function bigint!(int::Polymake.pm_IntegerAllocated)
+    unsafe_load(reinterpret(Ptr{BigInt},int.cpp_object))
```

Comment:
I am not sure we should really do this. It is not an inplace bigint, it returns a new object that shares memory with the old one. So doing
```
x = bigint!(x)
```
or
```
bigint!(pm_Integer(0))
```
will (not only can, but will!) cause a segfault.A better name would be `unsafe_convert_to_bigint`, which makes it a bit clearer that it should not be used with values that get potentially destroyed too early.

I am honestly not even sure the way you use it below is safe, I would have to investigate a bit.